### PR TITLE
✨feat: add privacy policy agreement

### PR DESCRIPTION
**Pull request by Marukome0743**
# Summary

- [x] Add privacy policy link to upload page
- [x] define `COMPANYS` constant value

## How to verify your code
I checked it in my Microsoft Edge browser.

### CODE REVIEW

File: components/profileForm.tsx [hunk 1]
```diff
@@ -107,13 +113,25 @@ export function ProfileForm() {
...
       </label>
-      <span className="after:ml-0.5 after:text-red-500 after:content-['*']">
-        個人情報提供への同意
-      </span>
-      <label className="label cursor-pointer">
-        <span className="label-text">同意する</span>
-        <input type="checkbox" className="checkbox" required={true} />
-      </label>
+      <div className="card bg-base-100 shadow-xl">
+        <div className="card-body items-center text-center">
```
Comment (kazutan1230) [inline components/profileForm.tsx:117]: Somehow, I felt strange that "個人情報提供への同意" was big.
I prefer this style how about you, but margin between a link and checkbox is too large.
How do you think about?
```
    <div className="card-body">
      <span className="after:ml-0.5 after:text-red-500 after:content-['*']">
        個人情報提供への同意
      </span>
      <Link
        href="https://www.openupgroup.co.jp/privacy-policy/"
        target="_blank"
        className="inline-flex gap-1 text-blue-600 underline"
      >
        個人情報保護方針
        <ArrowTopRightOnSquareIcon className="h-4 w-4" />
      </Link>
      <div className="card-body items-center">
        <label className="label cursor-pointer">
          <span className="label-text mr-2">同意する</span>
          <input type="checkbox" className="checkbox" required={true} />
        </label>
      </div>
```

File: components/profileForm.tsx [hunk 2]
```diff
@@ -107,13 +113,25 @@ export function ProfileForm() {
...
+            className="inline-flex items-center gap-1 text-blue-600 underline"
+          >
+            個人情報保護方針
+            <ArrowTopRightOnSquareIcon className="h-4 w-4" />
+          </Link>
+          <label className="label cursor-pointer">
+            <span className="label-text mr-2">同意する</span>
+            <input type="checkbox" className="checkbox" required={true} />
+          </label>
+        </div>
```
Comment (kazutan1230) [inline components/profileForm.tsx:133]: Somehow, I felt strange that "個人情報提供への同意" was big.
I prefer this style how about you, but margin between a link and checkbox is too large.
How do you think about?
```suggestion
        <div className="card-body">
          <span className="after:ml-0.5 after:text-red-500 after:content-['*']">
            個人情報提供への同意
          </span>
          <Link
            href="https://www.openupgroup.co.jp/privacy-policy/"
            target="_blank"
            className="inline-flex gap-1 text-blue-600 underline"
          >
            個人情報保護方針
            <ArrowTopRightOnSquareIcon className="h-4 w-4" />
          </Link>
          <div className="card-body items-center">
            <label className="label cursor-pointer">
              <span className="label-text mr-2">同意する</span>
              <input type="checkbox" className="checkbox" required={true} />
            </label>
          </div>
        </div>
```

**Review by kazutan1230 — COMMENTED** [review]
You don't use className card at "所属会社" and "障がい者手帳の画像・写真", do you?

File: components/profileForm.tsx (hunk 2 above)
Comment (Marukome0743) [reply components/profileForm.tsx]: I've fixed the title size.

**Comment by Marukome0743** [comment]
> You don't use className card at "所属会社" and "障がい者手帳の画像・写真", do you?

I have no plans of that.
I don't think the company and disability certificate image input are needs to change.

**Review by kazutan1230 — APPROVED** [review]
I approve this PR.

**Pull request merged**